### PR TITLE
Demo data

### PR DESCRIPTION
**Pull request by leigh-mil**
## Description
Mocked TO spending data so that active TOs have a portion of their budget spent and expired TOs only have a few dollars left. 
For A-Wing (Aardvark) and B-Wing (Beluga), the balance for the active TO matches the data mocked out for the reports. 
The names of all sample portfolios have been updated as well! 🚀
PR created in collaboration with @dandds 

## Pivotal
[https://www.pivotaltracker.com/story/show/164254277](https://www.pivotaltracker.com/story/show/164254277)

## Screenshots
screen shot 2019-03-01 at 3 13 40 pm
screen shot 2019-03-01 at 3 14 25 pm
screen shot 2019-03-01 at 3 14 40 pm

### CODE REVIEW

File: script/seed_sample.py [hunk 1]
```diff
@@ -92,166 +98,166 @@ def seed_db():
...
+        add_members_to_portfolio(portfolio, users=users)
+        add_active_task_order(portfolio)
+        add_expired_task_order(portfolio)
+        add_pending_task_order(portfolio)
+        add_applications_to_portfolio(portfolio, application_info)
 
 
 if __name__ == "__main__":
-    config = make_config({"DISABLE_CRL_CHECK": True})
+    config = make_config({"DISABLE_CRL_CHECK": True, "DEBUG": False})
```
Comment (drummond-dds) [inline script/seed_sample.py]: Is this just to suppress all the SQL logging output?

Comment (leigh-mil) [reply script/seed_sample.py]: Yes!

File: script/seed_sample.py [hunk 2]
```diff
@@ -92,166 +98,166 @@ def seed_db():
...
-    ]
-    for (start_date, end_date) in date_ranges:
-        task_order = TaskOrderFactory.build(
-            start_date=start_date,
-            end_date=end_date,
-            number=random_task_order_number(),
-            portfolio=funded_portfolio,
+    # create a portfolio 'Y-Wing' for each user
+    for user in users:
+        portfolio = Portfolios.create(
```
Comment (drummond-dds) [inline script/seed_sample.py]: We want every user to have a different portfolio with that same name? Might this cause confusion for testers thinking an update they made to a portfolio isn't showing up for other users?

Comment (leigh-mil) [reply script/seed_sample.py]: I asked Dan that, and he said to not worry about it for now. This PR is mostly for filling out the data in A-Wing and B-Wing.

**Comment by drummond-dds** [comment]
Looks good!

**Pull request merged**